### DISCUSSION TITLE
relocate nextcloud-aio

### DISCUSSION THREAD

**Discussion by nowatbuer** [nextcloud/all-in-one#4284] [Questions] [answered]
I am running a very small nextcloud-aio instance and want to move it to another domain. Via 
docker exec -it nextcloud-aio-nextcloud bash
 I logged into the nextcloud container and changed overwritehost trusted_domains overwrite.cli.url accordingly in config.php, After restarting all *aio* docker instances the changes were gone. Two questions in this context: 
Would it be sufficient to change these values?
Why are the changes gone after restart?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-domain

**Comment by nowatbuer** [comment]
Thanks, that helped. Haven't seen  that.
